test/Microsoft.Data.Analysis.Tests/Microsoft.Data.Analysis.Tests.csproj

```diff
@@ -27,5 +27,19 @@
             <AutoGen>True</AutoGen>
             <DependentUpon>DataFrameColumn.BinaryOperationTests.tt</DependentUpon>
         </Compile>
+        <Compile Update="Strings.Designer.cs">
```

Comment:
Not sure why, but I couldn't just link to the existing Strings.resx in src/Microsoft.Data.Analysis. But, I don't really care about having the extra Strings.resx file here, so I just added a new one. (This is needed because `TextFieldParser` is internal and we are including it in the test proj)

Reply:
I don't think we should be duplicating the files. You should be able to `Link` to the existing files. Let me know if you need help with this.

Reply:
Fixed. Thanks for the help!